src/main/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/infrastructure/client/FeignErrorDecoder.java

```diff
@@ -9,25 +11,62 @@
 import org.springframework.http.HttpStatus;
 import org.springframework.stereotype.Component;
 import org.springframework.web.server.ResponseStatusException;
+import uk.gov.hmcts.reform.iahomeofficeintegrationapi.domain.entities.HomeOfficeInstructResponse;
 
 @Component
 @Slf4j
 public class FeignErrorDecoder implements ErrorDecoder {
 
+    private final String errorLog = "Error in reading response body %s";
+
     @Override
     public Exception decode(String methodKey, Response response) {
 
         switch (response.status()) {
             case 400:
+                String errMessage = "";
+                try {
+                    if (response.body() != null && response.body().asInputStream() != null) {
+                        ObjectMapper mapper = new ObjectMapper();
+                        mapper.disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES);
```

Comment:
Any reason why you need another object mapper here?  Can't you use the project one in Spring config?

Reply:
changed it to use the Spring initiated bean